app/helpers/dashboard_helper.rb

```diff
@@ -6,22 +6,23 @@ def dashboard_cards_by_role
       challenge_manager: [
         { image_path: 'emoji_events', href: Rails.configuration.phx_interop[:phx_uri],
           alt: 'challenges', title: 'Challenges', subtitle: 'Create and manage challenges.' },
-        { image_path: 'people', href: 'manage_submissions',
+        { image_path: 'star_half', href: 'manage_submissions',
           alt: 'submissions', title: 'Submissions', subtitle: 'Manage submissions, evaluations, and evaluators.' },
-        { image_path: 'content_copy', href: 'evaluation_forms',
+        { image_path: 'check_circle_outline', href: 'evaluation_forms',
           alt: 'evaluation forms', title: 'Evaluation Forms', subtitle: 'Create and manage evaluation forms.' },
-        { image_path: 'map', href: 'user_guide',
-          alt: 'resources', title: 'Resources', subtitle: 'Learn how to make the most of the platform.' },
-        { image_path: 'support_agent', href: 'federal-agency-faqs',
-          alt: 'help', title: 'Help', subtitle: 'Get support on the Challenge.Gov platform.' }
+        { image_path: 'mail', href: "#{Rails.configuration.phx_interop[:phx_uri]}/messages",
+          alt: 'help', title: 'Message Center', subtitle: 'View and send messages to Challenge.gov users.' },
+        { image_path: 'assessment', href: "#{Rails.configuration.phx_interop[:phx_uri]}/analytics",
+          alt: 'help', title: 'Analytics', subtitle: 'View web analytics data related to your challenges.' },
+        { image_path: 'support', href: 'https://www.challenge.gov/cm-user-guide/',
+          alt: 'resources', title: 'Resources', subtitle: 'Learn how to make the most of the platform.' }
       ],
       evaluator: [
-        { image_path: 'content_copy', href: 'manage_submissions',
-          alt: 'submissions', title: 'Submissions', subtitle: 'Evaluate my assigned submissions.' },
-        { image_path: 'map', href: 'user_guide',
-          alt: 'user guides', title: 'Resources', subtitle: 'Learn how to make the most of the platform.' },
-        { image_path: 'support_agent', href: 'federal-agency-faqs',
-          alt: 'help', title: 'Help', subtitle: 'Get support on the Challenge.Gov platform.' }
+        { image_path: 'star_half', href: 'manage_submissions',
+          alt: 'submissions', title: 'Submissions',
+          subtitle: 'View submissions assigned to me and provide evaluations.' },
```

Comment:
Evaluators can't manage submissions, I think everything else is correct but the url (not sure how that slipped in before)
```suggestion
        { image_path: 'star_half', href: 'evaluations',
          alt: 'submissions', title: 'Submissions',
          subtitle: 'View submissions assigned to me and provide evaluations.' },
```